### ISSUE TITLE
Any examples for using node-usage for load-balancing?

### ISSUE THREAD

**Issue by smeijer** [meteorhacks/cluster#9]
Thank you for this great meteor package! I also noticed your great (a bit older) node library [node-usage](https://github.com/arunoda/node-usage).

So, is there any way of integrating these two? In the way that the system-resources are being taken into account for the balancing logic? I think that would be the ultimate goal, pointing the visitor to the most idle server, and ultimately alerting some system administrators when all servers become to heavily loaded (so he can spawn a new server).

**Comment by arunoda** [comment]
Yes that is. We might do something similar with integrating kadira. So, we
can use more resources.
We've not planned yet, but it'll be there for sure.
